packages/ticket_sidebar/src/components/Line.tsx

```diff
@@ -14,17 +14,26 @@ import {
   FlexContainer,
 } from './styled'
 
+const getUnit = (sellingMethod: SellingMethod) =>
+  sellingMethod === 'unit' ? 'uds' : 'kgs'
+
+const getPreparedQuantity = (prepared_quantity: number | undefined) =>
+  prepared_quantity ? prepared_quantity : '-'
```

Comment:
I think you can use nullish operator because the API returns null
```suggestion
const getPreparedQuantity = (prepared_quantity: number | undefined) =>
  prepared_quantity ?? '-'
```